tests/continuous/test_standard.py

```diff
@@ -813,3 +887,49 @@ def test_pbias_dask():
     result = result.compute()
     assert isinstance(result.data, np.ndarray)
     xr.testing.assert_equal(result, EXP_PBIAS3)
+
+
+@pytest.mark.parametrize(
+    ("fcst", "obs", "reduce_dims", "preserve_dims", "return_components", "scaling_factors", "expected"),
+    [
+        # Check reduce dim arg
+        (DA1_KGE, DA2_KGE, None, "space", False, None, EXP_KGE_KEEP_SPACE_DIM),
+        # Check preserve dim arg
+        (DA1_KGE, DA2_KGE, "time", None, False, None, EXP_KGE_KEEP_SPACE_DIM),
+        # Check reduce all
+        (DA3_KGE, DA2_KGE, None, None, False, None, EXP_KGE_REDUCE_ALL),
+        # returning components
+        (DA3_KGE, DA2_KGE, None, None, True, None, EXP_KGE_returns_components),
+        # Check scaling_factors
+        (DA3_KGE, DA2_KGE, None, None, False, [0.5, 1.0, 2.0], EXP_KGE_Scaling_Factors),
+        # Check different size arrays as input
+        (DA4_KGE, DA5_KGE, "space", None, False, None, EXP_KGE_DIFF_SIZE),
+    ],
+)
+def test_kge(fcst, obs, reduce_dims, preserve_dims, return_components, scaling_factors, expected):
+    """
+    Tests continuous.kge
+    """
+    result = scores.continuous.kge(
+        fcst,
+        obs,
+        reduce_dims=reduce_dims,
+        preserve_dims=preserve_dims,
+        return_components=return_components,
+        scaling_factors=scaling_factors,
+    )
+    # xr.testing.assert_equal(result, expected)
+    xr.testing.assert_allclose(result, expected, rtol=1e-10, atol=1e-10)
+
+
+def test_kge_dask():
+    """
+    Tests that continuous.kge works with Dask
+    """
+    fcst = DA3_KGE.chunk()
+    obs = DA2_KGE.chunk()
+    result = scores.continuous.kge(fcst, obs)
+    assert isinstance(result.data, dask.array.Array)  # type: ignore
+    result = result.compute()  # type: ignore
+    assert isinstance(result.data, np.ndarray)
```

Comment:
```suggestion
    assert isinstance(result.data, (np.ndarray, np.generic))
```
This change should help the unit tests pass on GitHub. It should make it compatible with different numpy versions.

Reply:
accepted into PR